pkg/trace/filters/validator.go

```diff
@@ -0,0 +1,28 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2021 Datadog, Inc.
+
+package filters
+
+import "github.com/DataDog/datadog-agent/pkg/trace/pb"
+
+// Validator holds a list of required tags.
+type Validator struct {
+	reqTags []string
+}
```

Comment:
Nice!

We also need a `rejectTags`, which rejects certain tags that have certain values.